documentation/modules/exploit/osx/local/vmware_fusion_lpe.md

```diff
@@ -0,0 +1,171 @@
+## Vulnerable Application
+
+This exploits an improper use of setuid binaries within VMware Fusion 10.1.3 - 11.5.2. The `Open VMware USB Arbitrator Service` can be
+launched outide of its standard path which allows loading of an attacker controlled binary. By creating a payload in the user home
+directory in a specific folder, and creating a hard link to the `Open VMware USB Arbitrator Service`, we're able to launch it
+temporarily to start our payload with an effective UID of 0.
+
+Additional description can be found in
+@mirchr's [exploit](https://raw.githubusercontent.com/mirchr/security-research/master/vulnerabilities/CVE-2020-3950.sh).
+
+It was found that VMware Fusion 11.5.3, which patched the previous vulnerability utilized an incomplete patch.  The patch checked
+for a correct code signature on the `VMware USB Arbitrator Service` at start, but not at launch, thus creating a TOCTOU race
+condition.  The discoverer @jeffball55 demoed the exploit working in ~30 attempts.  This module has been successful between
+5 and 25 attempts.
+
+VMware Fusion 11.5.1 is available from [VMware](https://download3.vmware.com/software/fusion/file/VMware-Fusion-11.5.1-15018442.dmg).
+
+## Verification Steps
+
+  1. Install the application
+  2. Start msfconsole
+  3. Get a shell
+  4. Do: ```use exploit/osx/local/vmware_fusion_lpe```
+  5. Do: ```set session #```
+  6. Do: ```run```
+  7. You should get a `euid=0` shell.
+
+## Options
+
+### MAXATTEMPTS
```

Comment:
Are we going with H3 for option names now? I've been bolding.

Reply:
@tperry-r7 changed it up a few weeks ago, its the biggest most recent change.

Reply:
Noted, thanks!